### PR TITLE
Added: Badge component

### PR DESCRIPTION
**Pull request by pratik-kumar-sahu**
- Added Badge component.
- Live link --> https://deploy-preview-3--fuzz-ui.netlify.app/pages/docs.html

### CODE REVIEW

File: styles/_components.css [hunk 1]
```diff
@@ -94,3 +91,71 @@
...
+    min-height: 5rem;
+    background-color: var(--color-primary-dark);
+}
+
+.code-snippet-container {
+    width: 100%;
+    min-height: 10rem;
+    max-height: 25rem;
+    padding: 1rem 1rem 0 1rem;
+    background-color: #2b3e50;
```
Comment (DivyanshSareen) [inline styles/_components.css]: use var for colors

File: styles/components/badge.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+.badge-online {
+    background-color: green;
+}
+
+.badge-offline {
+    background-color: var(--color-primary);
+}
+
+.badge-away {
+    background-color: #f59e0b;
```
Comment (DivyanshSareen) [inline styles/components/badge.css]: use vars

File: styles/_components.css (hunk 1 above)
Comment (pratik-kumar-sahu) [reply styles/_components.css]: Sure

File: styles/components/badge.css (hunk 2 above)
Comment (pratik-kumar-sahu) [reply styles/components/badge.css]: Sure

**Review by PrashanthreddyAinala — COMMENTED** [review]
be consistence while using colors, rest all looks good.

**Pull request merged**